monitoring/src/main/java/org/jlab/clas12/monitoring/tof_monitor.java

```diff
@@ -48,12 +48,15 @@ public class tof_monitor {
 	public H2F[][] DC_residuals_trkDoca;
 	public H1F[][] DC_residuals, DC_time, DC_time_even, DC_time_odd;
 	public H2F[][] DC_residuals_trkDoca_nocut;
-	public H1F[][] DC_residuals_nocut, DC_time_nocut;	
+	public H1F[][] DC_residuals_nocut, DC_time_nocut;
 	public H2F[][] DC_residuals_trkDoca_rescut;
 	public H1F[][] DC_residuals_rescut, DC_time_rescut;
+    public H2F DC_jitterzero_sec_sl, DC_jitterone_sec_sl, DC_jittertwo_sec_sl;
+    public H2F DC_hits_even_ts_sec_sl, DC_hits_odd_ts_sec_sl;
+    public H1F DC_jitterdist;
```

Comment:
Could you use `Enum` to reduce the repetition below?
```suggestion
    Enum DCenum = { jitterzero, jitterone, jittertwo, hits_even_ts, hits_odd_ts };
    HashMap<DCenum, H2F> DC_hits = new HashMap<>();
    public H1F DC_jitterdist;
```

Reply:
Yes we could do that in the next iteration